### PR TITLE
Validation test

### PR DESCRIPTION
**Pull request by KostyaTk**
Added unit tests for user data input validation functionality

### CODE REVIEW

File: src/test/java/ua/softserve/rv_018/greentourism/service/DataValidationUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	private String[] invalidPasswords =
+		{	"1234a", 	//Doesn't contain an upper case letter
+			"Abcde", 	//Doesn't contain a digit
+			"abcd1", 	//Doesn't contain a lower case letter
+			"Abcd 1",	//Contains whitespace
+			"Abc1",		//Contains not enough symbols
+			"#Abcd1",	//Contains inappropriate symbol
+			"&*#@`%" }; //Absolutely wrong password 
+	
+	private String[] validFLNames =
```
Comment (Sabfir) [inline src/test/java/ua/softserve/rv_018/greentourism/service/DataValidationUnitTest.java]: rename it to valideNames.
The same things for other variables with "FL.

File: src/test/java/ua/softserve/rv_018/greentourism/service/DataValidationUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
+package ua.softserve.rv_018.greentourism.service;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class DataValidationUnitTest {
+	private String[] validPasswords =
```
Comment (Sabfir) [inline src/test/java/ua/softserve/rv_018/greentourism/service/DataValidationUnitTest.java]: If you are not going to change the velue of this field, please, do it final.
The same thing for the other fields

File: src/main/java/ua/softserve/rv_018/greentourism/service/UserDataInputValidation.java [hunk 3]
```diff
@@ -10,51 +10,52 @@
...
+			+ "(?=.*[0-9])"			//A digit must occur at least once
 			+ "(?=.*[a-z])"			//A lower case letter must occur at least once
 			+ "(?=.*[A-Z])"			//An upper case letter must occur at least once
 			+ "[0-9a-zA-Z_-]"		//Only numerics, chars, underscores and hyphens allowed
-			+ "{5,}"			//Must contain at least 5 characters
-			+ "$";				//End of the string
+			+ "{5,}"				//Must contain at least 5 characters
+			+ "$";					//End of the string
 	
     private static final String FLNAME_PATTERN =
```
Comment (Sabfir) [inline src/main/java/ua/softserve/rv_018/greentourism/service/UserDataInputValidation.java]: plz, change FLNAME_PATTERN to something like NAME_PATTERN, FL is not readable.
Check this file for this FL and delete it, plz

**Comment by Sabfir** [comment]
reviewed

**Comment by ghost** [comment]
reviewed

**Pull request merged**